java/src/jmri/util/swing/FontComboUtil.java

```diff
@@ -102,45 +99,18 @@ public static synchronized void prepareFontLists() {
 
         // Initialise the font lists
         monospaced = new ArrayList<>();
+        monospaced.add("Monospaced");
         proportional = new ArrayList<>();
+        proportional.add("Serif");
         character = new ArrayList<>();
+        character.add("Monospaced");
         symbol = new ArrayList<>();
+        symbol.add("SansSerif");
         all = new ArrayList<>();
-
-        // Create a font render context to use for the comparison
-        Canvas c = new Canvas();
-        // Loop through all available font families
-        for (String s : GraphicsEnvironment.getLocalGraphicsEnvironment().getAvailableFontFamilyNames()) {
```

Comment:
This entire for block *must* be restored. It is used to separate the fonts by attributes of the fonts where that matters.

Reply:
Yesterday, I proposed something to make it optional, and it was finally agreed that it should be removed. To day I remove it, and have to finally restore it. I don't get it.